Content.Server/Atmos/Components/HeatExchangerComponent.cs

```diff
@@ -0,0 +1,40 @@
+namespace Content.Server.Atmos.Components;
+
+[RegisterComponent]
+public sealed class HeatExchangerComponent : Component
+{
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("inlet")]
+    public string InletName { get; set; } = "inlet";
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("outlet")]
+    public string OutletName { get; set; } = "outlet";
+
+    /// <summary>
+    /// Pipe conductivity (mols/kPa/sec).
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("g")]
+    public float G { get; set; } = 1f;
+
+    /// <summary>
+    /// Thermal convection coefficient (J/degK/sec).
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("k")]
+    public float K { get; set; } = 8000f;
+
+    /// <summary>
+    /// Thermal radiation coefficient. Number of "effective" tiles this
+    /// radiator radiates compared to superconductivity tile losses.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("alpha")]
+    public float alpha { get; set; } = 400f;
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("debug")]
+    public bool Debug { get; set; } = false;
```

Comment:
log calls should just be debug instead of info rather than this, i dont think we do this anywhere else

Reply:
If there are multiple heat exchangers on a map, this lets you select which heat exchanger you want to debug instead of spewing out logs for every single one. This came in handy a few times while debugging.

Reply:
seems like the kind of thing VV would be used for?

Reply:
Fair. I initially did this because I didn't want to fill up the component with a bunch of read-only debug values. I'll mull over whether I want to keep the debugging features and add them to the component or just remove them.

Reply:
fwiw I agree with mirror, debug data shouldn't be getting serialized.